src/app/page.tsx

```diff
@@ -62,14 +31,20 @@ export default function Home() {
       <div className="font-montserrat text-text">
         <Landing date={data.date} />
         <div className="max-w-screen-xl px-4 mx-auto my-12">
-          <div className="flex items-center justify-between w-1/3 my-8">
+          <div className="flex items-center justify-between w-2/3 my-8">
             <h2 className="text-3xl">Schedule</h2>
             <Link
               href="/talks"
               className="px-6 py-3 text-3xl duration-150 border border-b-2 rounded-full bg-primary/50 border-white/25 backdrop-blur-sm hover:scale-110 hover:border-text hover:-translate-y-2"
             >
               Talks
             </Link>
+            <Link
+              href="/past"
+              className="px-6 py-3 text-3xl duration-150 border border-b-2 rounded-full bg-primary/50 border-white/25 backdrop-blur-sm hover:scale-110 hover:border-text hover:-translate-y-2"
+            >
+              Past Talks
+            </Link>
```

Comment:
Nav doesn't look great on mobile.

Reply:
I think the immediate fix here is just to do a div wrapping the two Talks and Past Talks button and then do a flex and then md:flex-col. As is, the Past Talks button here overflows the page.

Reply:
removed this ... and put `past talks` where the tickets button went for now.